django/core/management/templates.py

```diff
@@ -81,9 +87,13 @@ def handle(self, app_or_project, name, target=None, **options):
                                    "exist, please create it first." % top_dir)
 
         extensions = tuple(handle_extensions(options['extensions']))
-        extra_files = []
-        for file in options['files']:
```

Comment:
I would put any refactoring of "files" handling in a separate commit to make it easier to see whether this enhancement affects that handling.

Reply:
Okay, I decided to undo it instead.